### PR TITLE
ZIOS-10134: Update input bar design for global timed messages 

### PR DESCRIPTION
**Pull request by alexaubry**
### Issues

We updated the design of the input bar to change these behaviours:

- Always show the ephemeral indicator when writing and a timeout is set
- Gray out and disable the ephemeral indicator when a global timeout is set
- Do not tint the icons when a timeout is set

We also fixed a bug where enabling Markdown would remove the "Timed message" placeholder.

### Solutions

- Use a stack view to display multiple right accessories next to the text field
- Use disabled-specific button images and colors

## Dependencies

- [ ] Property for global conversations timeout in Data Model 

## TODO
- [x] ~~Fix edit mode textView is compresed issue~~ (will fix layout in a new PR)

### CODE REVIEW

File: Wire-iOS Tests/InputBar/InputBarTests.swift [hunk 1]
```diff
@@ -133,26 +144,33 @@ class InputBarTests: ZMSnapshotTestCase {
...
     // Disabled until we figure out the `[MockUser conversationType]` crash after resetting the simulator / on CI
     func disabled_testThatItRendersCorrectlyInEditState() {
-        let sut = InputBar(buttons: buttons())
-        sut.translatesAutoresizingMaskIntoConstraints = false
-        sut.layer.speed = 0
-        sut.setInputBarState(.editing(originalText: "This text is being edited"), animated: false)
-        sut.updateFakeCursorVisibility()
-        CASStyler.default().styleItem(sut)
-        verifyInAllPhoneWidths(view: sut)
+        inputBar.layer.speed = 0
```
Comment (daehn) [inline Wire-iOS Tests/InputBar/InputBarTests.swift]: Can this one be enabled again?

**Comment by alexaubry** [comment]
@daehn Yeah, it's is normal. We now respect the value of `UIView.conversationLayoutMargins.right` for the right accessories.

File: Wire-iOS Tests/InputBar/InputBarTests.swift (hunk 1 above)
Comment (alexaubry) [reply Wire-iOS Tests/InputBar/InputBarTests.swift]: I'll try running them on CI

**Comment by daehn** [comment]
@alexaubry but now it's no longer centered with the icons below

**Comment by alexaubry** [comment]
@daehn Ah right, this is especially noticeable in edit mode, which I haven't tested. Will update the PR.

**Comment by billypchan** [comment]
The textfield's leading margin is modified?

https://github.com/wireapp/wire-ios/pull/2287/files?short_path=ec9555e#diff-ec9555eb5d3bcc9a186eb4d66be2c89a

**Comment by billypchan** [comment]
@alexaubry I updated this PR for the margins issue.

**Pull request merged**